app/components/ui/molecules/DashboardListItem.js

```diff
@@ -29,32 +30,44 @@ const dashboardDateText = date => {
 }
 
 const Root = styled(Flex)`
-  height: ${th('space.5')}
-  align-items: center;
-  padding: 0;
+  flex-direction: column;
   border-bottom: ${th('colorFurniture')};
   border-bottom-style: solid;
   border-bottom-width: 1px;
-  height: 107px;
+  justify-content: space-between;
+
+  ${media.mobileUp`
+    flex-direction: row;
+    align-items: flex-start;
+  `};
+  :hover {
+    color: ${th('colorPrimary')};
+  }
 `
 
 const TitleBox = styled(Box)`
   font-weight: bold;
   text-align: left;
-  font-size: 16px;
+  font-size: ${th('size.7')};
   flex-grow: 1;
+  padding: 0px ${th('space.3')} ${th('space.3')} 0px;
 `
-const DateBox = styled(Box)`
+const DateBox = styled(Flex)`
   color: ${th('colorTextSecondary')}
-  text-align: right;
-  padding-left: 24px;
-  flex: 0 0 120px;
+  justify-content: space-between;
+  padding: 0px 0px ${th('space.3')} 0px;
+
+  ${media.mobileUp`
+    flex-direction: column;
+    text-align: right;
+    flex: 0 0 120px;
+  `};
 `
-const RelativeDate = styled.p`
+const RelativeDate = styled(Box)`
   margin-bottom: 0;
```

Comment:
If you're changing this from a `p` to a `Box`, I don't think this line is necessary any more? I imagine that you only had this before, in order to remove the default margins that exist on `<p>` tags?